src/adagrams.js

```diff
@@ -1,15 +1,132 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  let letter_quantity = {
+    A: 9,
+    B: 2,
+    C: 2,
+    D: 4,
+    E: 12,
+    F: 2,
+    G: 3,
+    H: 2,
+    I: 9,
+    J: 1,
+    K: 1,
+    L: 4,
+    M: 2,
+    N: 6,
+    O: 8,
+    P: 2,
+    Q: 1,
+    R: 6,
+    S: 4,
+    T: 6,
+    U: 4,
+    V: 2,
+    W: 2,
+    X: 1,
+    Y: 2,
+    Z: 1,
+  };
+
+  let letter_bank = [];
+  const top_range = "A".charCodeAt(0);
+  const bottom_range = "Z".charCodeAt(0);
+
+  let counter = 0;
+  while (counter < 10) {
+    const rand_i = Math.random() * (top_range - bottom_range) + bottom_range;
```

Comment:
This has a very subtle bug.

Each letter drawn has a 1/26th chance of being drawn each time. So the odds of you drawing a Z on your first draw is the same as an E.